components/hab/src/command/studio/enter.rs

```diff
@@ -273,22 +272,30 @@ mod inner {
     }
 
     pub async fn start_windows_studio(ui: &mut UI, args: &[OsString]) -> Result<()> {
-        let command = match henv::var(super::STUDIO_CMD_ENVVAR) {
-            Ok(command) => PathBuf::from(command),
-            Err(_) => {
-                init()?;
-                let version: Vec<&str> = VERSION.split('/').collect();
-                let ident = PackageIdent::from_str(&format!("{}/{}",
-                                                            super::STUDIO_PACKAGE_IDENT,
-                                                            version[0]))?;
-                exec::command_from_min_pkg(ui, super::STUDIO_CMD, &ident).await?
-            }
-        };
-
-        if let Some(cmd) = find_command(command.to_string_lossy().as_ref()) {
-            process::become_command(cmd, args)?;
+        init()?;
+        let version: Vec<&str> = VERSION.split('/').collect();
+        let ident =
+            PackageIdent::from_str(&format!("{}/{}", super::STUDIO_PACKAGE_IDENT, version[0]))?;
+        let pwsh_command = exec::command_from_min_pkg(ui, "pwsh.exe", &ident).await?;
+        let studio_command = exec::command_from_min_pkg(ui, "hab-studio.ps1", &ident).await?;
+
+        let mut cmd_args: Vec<OsString> = vec!["-NoProfile".into(),
+                                               "-ExecutionPolicy".into(),
+                                               "bypass".into(),
+                                               "-NoLogo".into(),
+                                               "-File".into(),
+                                               studio_command.to_string_lossy().as_ref().into()];
```

Comment:
You could use `PathBuf::into_os_string()` here to simplify (or really just `.into()`).

Going further, I think you could probably do 
```
vec!["-NoProfile", 
     "-ExecutionPolicy", 
     "bypass", 
     "-NoLogo",
     "-File", 
     studio_command]
.into_iter()
.map(Into::into)
.collect();
```

Reply:
thanks! I knew there had to be a better way.